staging/src/k8s.io/mount-utils/go.mod

```diff
@@ -0,0 +1,16 @@
+// This is a generated file. Do not edit directly.
+
+module k8s.io/mount-utils
+
+go 1.13
```

Comment:
Does this need to be updated to 1.14?

Reply:
Current utils project do not need 1.14 but I do not see issues with bumping it up.

Reply:
It's 1.15 now :upside_down_face: 

https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apimachinery/go.mod#L5